lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -53,22 +53,20 @@ def load_graph(url:)
 
           access_end_dt = Time.now.utc
           @access_time_s = access_end_dt - access_start_dt
-          @fetched_size = full_graph.triples.to_s.size if performance_data?
           Rails.logger.info("Time to receive data from authority: #{access_time_s}s")
         end
 
         def normalize_results
           normalize_start_dt = Time.now.utc
 
-          @filtered_graph = graph_service.filter(graph: @full_graph, language: language)
+          @filtered_graph = graph_service.filter(graph: full_graph, language: language)
           results = map_results
           json = convert_results_to_json(results)
 
           normalize_end_dt = Time.now.utc
           @normalize_time_s = normalize_end_dt - normalize_start_dt
-          @normalized_size = json.to_s.size if performance_data?
           Rails.logger.info("Time to normalize data: #{normalize_time_s}s")
-          json = append_performance_data(json) if performance_data?
+          json = append_data_outside_results(json)
```

Comment:
Generalizing the name of this method so it can be used later to add other types of data to the results.